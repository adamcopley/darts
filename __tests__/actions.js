import {
    onScoreChange,
    onScoreSubmit,
    onScoreClear,
    onScoreUndo,
    onGameStart,
    onGameStartNew,
    onGameSetName,
    onGameSetTotal
} from '../app/actions'

describe('Actions snapshots', () => {
    test('onScoreChange', () => {
        expect(onScoreChange()).toMatchSnapshot()
    })

    test('onScoreSubmit', () => {
        expect(onScoreSubmit()).toMatchSnapshot()
    })

    test('onScoreClear', () => {
        expect(onScoreClear()).toMatchSnapshot()
    })

    test('onScoreUndo', () => {
        expect(onScoreUndo()).toMatchSnapshot()
    })

    test('onGameStart', () => {
        expect(onGameStart()).toMatchSnapshot()
    })

    test('onGameStartNew', () => {
        expect(onGameStartNew()).toMatchSnapshot()
    })

    test('onGameSetName', () => {
        expect(onGameStartNew()).toMatchSnapshot()
    })

    test('onGameSetTotal', () => {
        expect(onGameStartNew()).toMatchSnapshot()
    })
})