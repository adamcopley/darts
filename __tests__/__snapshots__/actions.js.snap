// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions snapshots onGameSetName 1`] = `
Object {
  "type": "GAME_START_NEW",
}
`;

exports[`Actions snapshots onGameSetTotal 1`] = `
Object {
  "type": "GAME_START_NEW",
}
`;

exports[`Actions snapshots onGameStart 1`] = `
Object {
  "type": "GAME_START",
}
`;

exports[`Actions snapshots onGameStartNew 1`] = `
Object {
  "type": "GAME_START_NEW",
}
`;

exports[`Actions snapshots onScoreChange 1`] = `
Object {
  "type": "SCORE_CHANGE",
}
`;

exports[`Actions snapshots onScoreClear 1`] = `
Object {
  "type": "SCORE_CLEAR",
}
`;

exports[`Actions snapshots onScoreSubmit 1`] = `
Object {
  "type": "SCORE_SUBMIT",
}
`;

exports[`Actions snapshots onScoreUndo 1`] = `
Object {
  "type": "SCORE_UNDO",
}
`;
